doc/user_guide/visualisation.rst

```diff
@@ -1332,3 +1332,25 @@ Permanent markers are stored in the HDF5 file if the signal is saved:
     >>> s1 = hs.load("storing_marker.hdf5")
     >>> s1.metadata.Markers
     └── point = <hyperspy.drawing._markers.point.Point object at 0x7efcfadb06d8>
+
+Markers have 'zorder' property to support ordering between markers.
```

Comment:
If you want to add some code highlighting, as it is usually done:
```suggestion
Markers have ``'zorder'`` property to support ordering between markers.
```

Reply:
fixed. "'"'s are replaced with "``".